libbeat/docs/processors-config.asciidoc

```diff
@@ -359,6 +360,22 @@ _Tencent Cloud_
 }
 -------------------------------------------------------------------------------
 
+_Alibaba Cloud_
+
```

Comment:
I realize that this is supposed to just be examples of the output, but it would be good to mention something like, "Only when VPC is selected as the network type of the instance, you can get the metadata."

@dedemorton Maybe you have wording or formatting suggestions.